lang/java/reef-examples/src/main/java/org/apache/reef/examples/hello/HelloReefAzBatch.java

```diff
@@ -54,13 +59,22 @@ private static Configuration getEnvironmentConfiguration() throws IOException {
    *
    * @return the configuration of the HelloREEF driver.
    */
-  private static Configuration getDriverConfiguration() {
-    return DriverConfiguration.CONF
+  private static Configuration getDriverConfiguration(boolean includeContainerConfiguration) {
+    Configuration driverConfiguration = DriverConfiguration.CONF
         .set(DriverConfiguration.DRIVER_IDENTIFIER, "HelloREEF")
         .set(DriverConfiguration.GLOBAL_LIBRARIES, EnvironmentUtils.getClassLocation(HelloDriver.class))
         .set(DriverConfiguration.ON_DRIVER_STARTED, HelloDriver.StartHandler.class)
         .set(DriverConfiguration.ON_EVALUATOR_ALLOCATED, HelloDriver.EvaluatorAllocatedHandler.class)
         .build();
+
+    if (!includeContainerConfiguration) {
+      return driverConfiguration;
```

Comment:
Synced offline. See if we really needs this or just binding in AzureBatchBootStrapLauncher.

Reply:
Removed binding here.

---
In reply to: [201107846](https://github.com/sharathmalladi/reef/pull/5#discussion_r201107846) [](ancestors = 201107846)